project/AppDependencies.scala

```diff
@@ -11,6 +10,7 @@ object AppDependencies {
   val hmrcMongoVersion  = "0.55.0"
 
   val compile = Seq(
+    "org.reactivemongo"      %% "play2-reactivemongo"       % "0.20.13-play28",
```

Comment:
Please do not do this - this service uses hmrc-mongo because reactivemongo is deprecated